### PR TITLE
git-appraise 0.6 (new formula)

### PR DESCRIPTION
**Pull request by stephengroat**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

add git-appraise formula

### CODE REVIEW

File: Formula/git-appraise.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/google").mkpath
+    ln_s buildpath, buildpath/"src/github.com/google/git-appraise"
+
+    system "go", "build", "-o", bin/"git-appraise", "./src/github.com/google/git-appraise/git-appraise"
```
Comment (ilovezfs) [inline Formula/git-appraise.rb]: Have you tried building without `, "./src/github.com/google/git-appraise/git-appraise"`?

File: Formula/git-appraise.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  test do
+    system bin/"git-appraise", "help"
```
Comment (ilovezfs) [inline Formula/git-appraise.rb]: This needs to be a test that exercises and verifies the functionality of the software.

Comment (ilovezfs) [inline Formula/git-appraise.rb]: This needs to be a test that exercises and verifies the functionality of the software.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/git-appraise.rb (hunk 1 above)
Comment (stephengroat) [reply Formula/git-appraise.rb]: doesn't work, need to direct the binary to a specific location

Comment (ilovezfs) [reply Formula/git-appraise.rb]: You should be able to make it `"github.com/google/git-appraise/git-appraise"`

Comment (stephengroat) [reply Formula/git-appraise.rb]: done

File: Formula/git-appraise.rb (hunk 2 above)
Comment (stephengroat) [reply Formula/git-appraise.rb]: kinda like `git-lfs`, if i do significant testing, i'll need `git`.

should i make `git` a `depends` or `recommends` then? it's not listed under `git-lfs`, which is kinda what confused me

Comment (DomT4) [reply Formula/git-appraise.rb]: `git` ships with macOS, so, no dependency required to use it.

Comment (stephengroat) [reply Formula/git-appraise.rb]: done

**Comment by stephengroat** [comment]
should be good and ready for review

File: Formula/git-appraise.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    (testpath/"README").write "test"
+    system "git", "add", "README"
+    system "git", "commit", "-m", "Init"
+    system "git", "branch", "user/test"
+    system "git", "checkout", "user/test"
+    (testpath/"README").append_lines "test2"
+    system "git", "add", "README"
+    system "git", "commit", "-m", "Update"
+    system "git", "appraise", "request", "--allow-uncommitted"
+    assert_predicate (testpath/".git/refs/notes/devtools/reviews"), :exist?
```
Comment (DomT4) [inline Formula/git-appraise.rb]: The `()` shouldn't be needed here.

Comment (DomT4) [reply Formula/git-appraise.rb]: They're not harmful or anything, we just haven't used them with many other `assert_predicate` statements thus far & consistency is nice.

Comment (stephengroat) [reply Formula/git-appraise.rb]: @DomT4 just got lazy, cleaned up

**Comment by stephengroat** [comment]
should be good to go and ready for review

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
Thanks for the new formula, @stephengroat! You're awesome :)

**Pull request closed without merging**